src/Faker/Provider/hi_IN/Company.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
```

Comment:
the lines are generally too longs, it makes diffs difficult to read. Please split your lines either by capital letter, or at about 80 characters.